.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+    "env": {
+        "node": true,
+        "es6": true
+    },
+    "extends": "eslint:recommended",
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly",
+        "document": "readonly",
+        "describe": "readonly",
+        "test": "readonly",
+        "expect": "readonly",
+        "history": "readonly"
+    },
+    "parserOptions": {
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "rules": {
+        "semi": ["warn", "always"],
+        "quotes": ["warn", "single"]
+    },
+    "root": true
+}
```

Comment:
Could you set your editor to add trailing newlines?  It's already set up in our `.editorconfig` file if you want to configure your editor to use that.